creepy/query/__init__.py

```diff
@@ -151,16 +160,29 @@ def download(self, obj: ProxyObject):
         return self._post(DownloadQuery(obj._id))
 
     def _send_file(self, src_path: str, dst_path: str, exist_ok=False):
-        CHUNK_SIZE = 2**20
+        CHUNK_SIZE = 2**24
```

Comment:
Why do you change this value so often? :)

Reply:
Are you sure? It's second time I change it.

Reply:
I definitely remember that you changed 2^20 -> 2^24 in some previous commit. AFAIK, it was not the first time.

Reply:
First time I changed 24->20, this time I changed 20->24, that's it)